unstructured/chunking/__init__.py

```diff
@@ -19,82 +19,75 @@
 _P = ParamSpec("_P")
 
 
-def add_chunking_strategy() -> Callable[[Callable[_P, List[Element]]], Callable[_P, List[Element]]]:
+def add_chunking_strategy(func: Callable[_P, List[Element]]) -> Callable[_P, List[Element]]:
```

Comment:
This looks like a lot of changes but it's essentially just sliding the body out by one indent level.